spec/importers/file_importers/attendance_importer_spec.rb

```diff
@@ -63,6 +73,33 @@
         end
       end
 
+      context 'row with absence (district does not have dismissed/reason/excused/comment fields)' do
```

Comment:
I don't want to make adding these new data columns / features dependent on New Bedford. My guess is that New Bedford also has fields that map to dismissed/reason/excused/comment, but they might not, or their data structure might be different. 

Either way, we can't control how quickly or slowly we'd be able to get those columns exported for New Bedford. So I want to make sure that changing the database structure and import code won't break anything on the New Bedford side even if their incoming CSV rows have a different shape. The net effect of this change for New Bedford should be that their absences/tardies tables each have four additional empty columns.

Reply:
This might be a good thing to add as a method in`PerDistrict`, since we want to intentionally treat this differently across districts.